docs/using_cameras.md

```diff
@@ -0,0 +1,98 @@
+
+# Using Cameras with ILLIXR
+
+ILLIXR supports a wide range of cameras, most of which are for the purpose of feeding images and IMU measurements to the system. <br>
+*To learn more about how it works, checkout [Building ILLIXR][12].* <br>
+*If you are interested in what topics these cameras feed into, checkout [ILLIXR plugins][10].*
+
+
+**Important Note**: <br>
+Before running any of these plugins below, it is important to comment out `offline_imu_cam` in `configs\native.yaml`. <br>
+
+## Add Calibration Parameters
+In order to add your camera's calibration extrinsics, you need to modify your choice of SLAM/VIO plugin. Follow these instructions to [modify a plugin][21].
+
+-   [`Kimera-VIO`][5]:
+    
+    Navigate to `params`. <br>
+    Add your calibration here. Look at the other folders for reference. <br>
+    Modify [this line][7] in `examples/plugin.cpp` to point to your calibration folder. 
+
+-   [`OpenVINS`][6]:
+    
+    Navigate to `ov_msckf/src`.<br>
+    Uncomment [this line][3] out in `slam2.cpp` in order to use ZED's calibration parameter for OpenVins. <br>
+    You can add your own calibration parameters in the same file.     
+
+## ZED Mini
+
+1. **Install ZED SDK** 
+
+    Install the latest version of the ZED SDK on [stereolabs.com][1].<br>
+    For more information, checkout the [ZED API documentation][2].
+
+2. **Get ZED's calibration parameters**
+
+    Both OpenVINS and Kimera have a decent calirbation parameters for ZED. But if you wish to add your own: 
+    
+        /usr/local/zed/tools/ZED_Calibration
+
+    Your original factory calibration file is stored here
+
+        /usr/local/zed/settings/
+
+    Or download it from [calib.stereolabs.com][8].
+
+3. **Run ILLIXR with ZED:**  
+
+    Uncomment `zed` in `configs/native.yaml` and run ILLIXR normally.
```

Comment:
You also need to update the flag in VIO right?

Reply:
zed also does not exist in configs/native.yaml

Reply:
You're right. It's in `configs/rt_slam_plugins/yaml`. My bad.